src/components/Character/character.test.jsx

```diff
@@ -0,0 +1,9 @@
+import { render } from "@testing-library/react";
+import Character from "./Character";
+
+it("should render the Character component", () => {
+	const { container } = render(
+		<Character head='boar' middle='blue' bottom='blue' />
+	);
+	expect(container).toMatchSnapshot();
```

Comment:
You can add additional expect statements here to look for particular things on the page like images, etc.